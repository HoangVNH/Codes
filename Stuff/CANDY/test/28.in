10
25
VVXVVDDDDVXXXXDDDDDDDXVVV
47
VVVDDVVVVVVVDDDXXXXXXXXXXXVVVVVDDDXXXXXVVVVDDDD
49
VVVVVVVVXXXXXDDDDDDDDVVDDXXXXXXXXXXXXXVVVVXXVVDDX
28
DDDDVVVVVXXXXXXXXDDDXXVVVVVV
36
VVVVVVVVXXXVVXXXXXDVVVVVDDDVVVVXXXXX
22
XXXXXXXXXVDDXXXVVVVVVV
33
XDVVVVVVXXXXXDDXXXDDDDVVVVVVVVVVD
24
XXVDDDXVVVDDDDDDDDDDDDDD
26
DDVVVVDDDDVVXDDDDDDXDDDDDD
22
VVVXXXXXXXDDXXXXXXXXXX
