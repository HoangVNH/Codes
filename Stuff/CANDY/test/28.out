24
46
48
27
35
20
32
23
24
21
